wiki/Article_Styling_Criteria/en.md

```diff
@@ -867,34 +770,38 @@ When writing the name of the game modes, they must be written as follows:
 - `osu!catch`
 - `osu!mania`
 
-*Notice: To maintain consistency, use `osu!standard` when referring to the game mode. Folder names, however, must use `osu!` (not `osu!standard`) even if it is referring to the game mode.*
+*Notice: `osu!standard` is used to maintain consistency when referring to the game mode. Folder names, however, must use `osu!` even if it is referring to the game mode.*
+
+Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) can be done if discussing about said game mode's previous name.
 
-You may only refer to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) only when discussing about said game mode's previous name.
+Articles that are more official, e.g. [Ranking Criteria](/wiki/Ranking_Criteria), may use `osu!` instead of `osu!standard`.
 
 ### Terminology
 
-These words are spelt as follows (note the spacing):
+These words are spelt as follows (spacing must match):
 
-- `approach circles`
+- `approach circle`
 - `game mode` (or `mode`)
 - `game mod` (or `mod`)
 - `gameplay`
 - `hit burst`
-- `hit circles`
-- `hit sounds`
+- `hit circle`
+- `hit sound`
 - `in-game`
 - `play style`
```

Comment:
Either use the suggestion below (idk why it does this on non-diffs) or commit another way.
```suggestion
- `playstyle`
```

Reply:
We are going with "playstyle" now?

Reply:
Never mind, looked back at the Discord again.